src/app.controller.put.ts

```
@@ -147,4 +147,174 @@
+						x_request_id,
+					)) as FindOneResponseObject
+					await this.websockets.publish(schema, SocketType.UPDATE, result[schema.primary_key])
+					successful++
+					data.push(result)
+				} catch (e) {
+					errored++
+					errors.push({
+						item: body.indexOf(item),
+						message: e.message,
+					})
+					continue
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)